ClosedXML/Excel/XLWorkbook.cs

```diff
@@ -52,6 +52,14 @@ public static IXLStyle DefaultStyle
             }
         }
 
+        internal static XLStyleValue DefaultStyleValue
```

Comment:
Note this new property in `XLWorkbook`.

Reply:
Nice idea, though 
```get { return XLStyleValue.Default; }```
would be better, since ```XLStyle.Default``` creates a new instance each time.

Reply:
Oh, I expected `XLStyle.Default` to be created only once. Is that not possible?

Reply:
It is possible, but one can modify this value afterward, which is undesirable.